diff -durN atop-1.6.orig/acctproc.c atop-1.6/acctproc.c
--- atop-1.6.orig/acctproc.c	Tue Sep  3 10:03:33 2002
+++ atop-1.6/acctproc.c	Tue Sep 17 14:17:57 2002
@@ -55,6 +55,7 @@
 
 #include <sys/types.h>
 #include <stdio.h>
+#include <string.h>
 #include <fcntl.h>
 #include <errno.h>
 #include <unistd.h>
@@ -76,7 +77,11 @@
 static	uint	acctsize;	/* previous size of account file	  */
 int   		acctfd   = -1;	/* fd of account file       		  */
 
+#ifdef __alpha__
+static long	acctexp(time_t ct);
+#else
 static long 	acctexp(comp_t ct);
+#endif
 
 /*
 ** Semaphore-handling
@@ -419,8 +424,13 @@
 		api->cpu.utime  = acctexp(acctrec.ac_utime);
 		api->mem.minflt = acctexp(acctrec.ac_minflt);
 		api->mem.majflt = acctexp(acctrec.ac_majflt);
+#ifdef __alpha
+		api->mem.iosw	= 0;
+		api->dsk.ior	= 0;
+#else
 		api->mem.iosw   = acctexp(acctrec.ac_swaps);
 		api->dsk.ior    = acctexp(acctrec.ac_rw);
+#endif
 
 		strcpy(api->gen.name, acctrec.ac_comm);
 	}
@@ -433,8 +443,13 @@
 /*
 ** expand the special compression-method
 */
+#ifdef __alpha
+long
+acctexp(time_t ct)
+#else
 long
 acctexp(comp_t ct)
+#endif
 {
         register long	e;
         register long	f;
