diff -ur atop-1.10.orig/acctproc.c atop-1.10/acctproc.c
--- atop-1.10.orig/acctproc.c	2003-07-16 02:03:08.000000000 +0200
+++ atop-1.10/acctproc.c	2003-07-16 02:08:24.000000000 +0200
@@ -98,7 +98,11 @@
 static	off_t	acctsize;	/* previous size of account file	  */
 int   		acctfd   = -1;	/* fd of account file       		  */
 
-static count_t 	acctexp(comp_t ct);
+#ifdef __alpha__
+static count_t	acctexp(time_t ct);
+#else
+static count_t	acctexp(comp_t ct);
+#endif
 
 /*
 ** Semaphore-handling
@@ -439,8 +443,13 @@
 		api->cpu.utime  = acctexp(acctrec.ac_utime);
 		api->mem.minflt = acctexp(acctrec.ac_minflt);
 		api->mem.majflt = acctexp(acctrec.ac_majflt);
+#ifdef __alpha
+		api->mem.iosw   = 0;
+		api->dsk.ior    = 0;
+#else
 		api->mem.iosw   = acctexp(acctrec.ac_swaps);
 		api->dsk.ior    = acctexp(acctrec.ac_rw);
+#endif
 
 		strcpy(api->gen.name, acctrec.ac_comm);
 	}
@@ -453,8 +462,13 @@
 /*
 ** expand the special compression-method
 */
+#ifdef __alpha
+static count_t
+acctexp(time_t ct)
+#else
 static count_t
 acctexp(comp_t ct)
+#endif
 {
         count_t	exp;
         count_t val;
Tylko w atop-1.10: acctproc.c~
